testsuite/openshift/objects/envoy/tls.py

```diff
@@ -0,0 +1,85 @@
+"""Envoy Gateway implementation with TLS setup"""
+from importlib import resources
+from typing import TYPE_CHECKING
+
+import yaml
+
+from . import Envoy, EnvoyConfig
+
+if TYPE_CHECKING:
+    from ...client import OpenShiftClient
+
+TLS_TRANSPORT = """
```

Comment:
Why you moved this from yaml file to constant?

Reply:
Les duplicatiom, the file contained all the resources, this only relevant sections